docs/usage/CATEGORY_SYNC.md

```diff
@@ -69,17 +69,17 @@ events.
 
 - `beforeUpdateCallback`
 a filter function which can be applied on a generated list of update actions. It allows the user to intercept category 
-update and modify (add/remove) update actions just before they are send to CTP API.
+update and modify (add/remove) update actions just before they are sent to CTP API.
 
 - `beforeCreateCallback`
 a filter function which can be applied on a category draft before a request to create it on CTP is issued. It allows the 
-user to intercept category create requests modify the draft before the create request is sent to CTP API.
+user to intercept category creates requests to modify the draft before the create request is sent to CTP API.
```

Comment:
It's supposed to be _category create_(noun) requests. Not the verb: create

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/2c2b61de5ac9d3d91b5f139a71ae8c13775c2e9a

Reply:
nitpick: maybe make create requests bold and/or italic ->  **_create requests_** as well as **_update requests_**

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319#discussion_r225940333